internal/controllers/machinedeployment/machinedeployment_sync.go

```diff
@@ -324,6 +324,7 @@ func (r *Reconciler) computeDesiredMachineSet(ctx context.Context, deployment *c
 	} else {
```

Comment:
Let's please chat via Zoom about the impact of the new field on KCP

As far as I can tell as of now it simply doesn't exist in the KCP object, which seems like a good state for the current PR

Reply:
Chatter, related work is already in the tracking issue